drivers/interrupt_controller/Kconfig.gic

```diff
@@ -36,4 +36,12 @@ config GIC_VER
 	default 2 if GIC_V2
 	default 3 if GIC_V3
 
+config GIC_SINGLE_SECURITY_STATE
+	bool
+	depends on GIC_V3
+	default n
```

Comment:
configs default to n so this is redundant

Reply:
OK, I'll remove it